camlp4/Camlp4/Struct/Grammar/Print.ml

```diff
@@ -70,7 +70,7 @@ module Make (Structure : Structure.S) = struct
   and print_rule ppf symbols =
     do {
       fprintf ppf "@[<hov 0>";
-      let _ =
+      let _ : Format.formatter -> unit =
```

Comment:
This is a bug, right? The code was written to print the rules, but in fact it prints nothing because a formatter argument is missing.

(The code is written in a rather hard to understand way, I guess that we could use `Format.pp_print_list` instead.)